src/services/expenses.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const expenses = [];
+
+const getExpenses = (searchParams) => {
+  const {
+    userId,
+    categories,
+    from,
+    to,
+  } = searchParams;
+
+  let foundExpenses = expenses;
+
+  if (userId) {
+    foundExpenses = expenses.filter(expense => expense.userId === +userId);
```

Comment:
The search function is not working properly. The 'foundExpenses' value should be replaced with the modified 'foundExpenses' value from the previous operation, not the original 'expenses' array.